google-apps-script.js

```diff
@@ -21,14 +25,23 @@ function doPost(e) {
   try {
     Logger.log(e); // the Google Script version of console.log see: Class Logger
     record_data(e);
-
-    var mailData = e.parameters; // just create a slightly nicer variable name for the data
+    
+    // shorter name for form data
+    var mailData = e.parameters;
+    
+    // names and order of form elements
+    var dataOrder = JSON.parse(e.parameters.formDataNameOrder);
+    
+    // determine recepient of the email
+    // if you have your email uncommented above, it uses that `TO_ADDRESS`
+    // otherwise, it defaults to the email provided by the form's data attribute
+    var sendEmailTo = (typeof TO_ADDRESS !== "undefined") ? TO_ADDRESS : mailData.formGoogleSendEmail;
```

Comment:
the only "_danger_" of allowing people to specify an email address to _send_ email to in the form is that it can become a vehicle for spam/phishing. 
(_I agree with it as a feature in principal, just in practice I've seen it abused_)